designs/index.md

```diff
@@ -53,7 +63,6 @@ or [using Markdown](#using-markdown). Read below about for a
 Proposals that have a user-visible impact must have a section documenting the
 impact on backward compatibility (and a rollout plan if needed).
 
-
 ### Create a Pull Request
 
 The author creates a Pull Request (PR) to add the document to
```

Comment:
This section starts using "the author" instead of "you". Is there a reason for that? Can we continue to directly address the contributor?

Reply:
Rephrased.